### PR TITLE
First attempt at documenting colander.drop

### PR DESCRIPTION
**Pull request by lmctv**

### CODE REVIEW

File: colander/__init__.py [hunk 1]
```diff
@@ -1851,8 +1852,10 @@ class _SchemaNode(object):
     - ``typ``: The 'type' for this node can optionally be passed in as a
       keyword argument. See the documentation for the positional arg above.
 
-    - ``default``: The default serialization value for this node.
-      Default: :attr:`colander.null`.
+    - ``default``: The default serialization value for this node when
+      not set.  If ``default`` is :attr:`colander.drop`, the node
+      will be dropped from schema serialization.  If not provided,
+      the node will be serialized to :attr:`colander.null` if not set.
```
Comment (stevepiercy) [inline colander/__init__.py]: This last sentence has repetitive clauses, "If not provided" and "if not set".

**Comment by stevepiercy** [comment]
Other than my comment, LGTM.

Does the requirement for signing https://github.com/Pylons/colander/blob/master/CONTRIBUTORS.txt apply for minor doc amendments?

**Comment by lmctv** [comment]
Amended commit to remove the duplication. As for CONTRIBUTORS.txt, my signature is already there at line 112. Thank you.

**Comment by stevepiercy** [comment]
LGTM. @mmerickel I'll let you handle merge since I know you're working on cleaning up the repo, per https://github.com/Pylons/colander/issues/246#issuecomment-172368662

**Pull request merged**